src/anyio/_backends/_asyncio.py

```diff
@@ -141,7 +141,9 @@ def find_root_task() -> asyncio.Task:
         for task in all_tasks():
             if task._callbacks:
                 for cb in _get_task_callbacks(task):
-                    if cb is _run_until_complete_cb or cb.__module__ == 'uvloop.loop':
+                    if isinstance(cb, partial):
+                        cb = cb.func
```

Comment:
we know that `_run_until_complete_cb` won't be used with partial, and 'uvloop.loop' doesn't use it either

Reply:
Good point, I updated to `continue` the for loop if it's a partial.